core/src/main/resources/i18n/displayStrings_tr.properties

```diff
@@ -0,0 +1,3345 @@
+# suppress inspection "TrailingSpacesInProperty"
+portfolio.pending.step3_seller.onPaymentReceived.name=Lütfen ayrıca ticaret sözleşmesinde belirtilen gönderici adının banka ekstrenizde görünen adla eşleştiğini doğrulayın:\nTicaret sözleşmesine göre gönderici adı: {0}\n\nİsimler tam olarak aynı değilse, ödeme alımını onaylamayın. Bunun yerine, "alt + o" veya "option + o" tuşlarına basarak bir anlaşmazlık açın.\n\n
+# suppress inspection "TrailingSpacesInProperty"
+portfolio.pending.step3_seller.onPaymentReceived.note=Lütfen unutmayın, ödeme alımını onayladığınız anda, kilitli ticaret tutarı XMR alıcısına serbest bırakılacak ve güvenlik depozitosu iade edilecektir.\n\n
+portfolio.pending.step3_seller.onPaymentReceived.confirm.headline=Ödemeyi aldığınızı onaylayın
+portfolio.pending.step3_seller.onPaymentReceived.confirm.yes=Evet, ödemeyi aldım
+portfolio.pending.step3_seller.onPaymentReceived.signer=ÖNEMLİ: Ödeme alımını onaylayarak, karşı tarafın hesabını da \
+  doğrulamış ve buna göre imzalamış olursunuz. Karşı tarafın hesabı henüz imzalanmadığından, geri ödeme riskini azaltmak için \
+  ödeme onayını mümkün olduğunca geciktirmelisiniz.
+
+portfolio.pending.step5_buyer.groupTitle=Tamamlanan ticaret özeti
+portfolio.pending.step5_buyer.groupTitle.mediated=Bu ticaret arabuluculukla çözüldü
+portfolio.pending.step5_buyer.groupTitle.arbitrated=Bu ticaret hakemlikle çözüldü
+portfolio.pending.step5_buyer.tradeFee=Ticaret ücreti
+portfolio.pending.step5_buyer.makersMiningFee=Madencilik ücreti
+portfolio.pending.step5_buyer.takersMiningFee=Toplam madencilik ücretleri
+portfolio.pending.step5_buyer.refunded=İade edilen güvenlik depozitosu
+portfolio.pending.step5_buyer.amountTooLow=Transfer edilecek tutar, işlem ücretinden ve mümkün olan minimum işlem değerinden (toz) daha düşük.
+portfolio.pending.step5_buyer.tradeCompleted.headline=Ticaret tamamlandı
+portfolio.pending.step5_buyer.tradeCompleted.msg=Tamamlanan ticaretleriniz \"Portföy/Tarihçe\" altında saklanır.\nTüm Monero işlemlerinizi \"Fonlar/İşlemler\" altında inceleyebilirsiniz.
+portfolio.pending.step5_buyer.bought=Satın aldınız
+portfolio.pending.step5_buyer.paid=Ödediniz
+
+portfolio.pending.step5_seller.sold=Satış yaptınız
+portfolio.pending.step5_seller.received=Aldınız
+
+tradeFeedbackWindow.title=Ticaretiniz başarıyla tamamlandı!
+tradeFeedbackWindow.msg.part1=Deneyiminiz hakkında geri bildirim almak isteriz. Yazılımı geliştirmemize ve pürüzleri gidermemize yardımcı olacaktır.
+tradeFeedbackWindow.msg.part2=Herhangi bir sorunuz varsa veya herhangi bir sorun yaşadıysanız, diğer kullanıcılar ve katkıda bulunanlarla Matrix sohbet odamızda iletişime geçin:
+tradeFeedbackWindow.msg.part3=Haveno'yu kullandığınız için teşekkürler!
+
+portfolio.pending.role=Rolüm
+portfolio.pending.tradeInformation=Ticaret bilgileri
+portfolio.pending.remainingTime=Kalan süre
+portfolio.pending.remainingTimeDetail={0} (kadar {1})

```

Comment:
it should be ``{0} ({1} tarihine kadar)``

Reply:
thanks, I made mistakes in some places, I would appreciate if you share more translation mistakes below this post. I will collect them all over time and open a new pull request